app/Commands/LaunchCommand.php

```diff
@@ -107,7 +110,43 @@ public function schedule(Schedule $schedule): void
         // $schedule->command(static::class)->everyMinute();
     }
 
-    private function createApp($appName): string
+    private function getOrganizationName() : string
+    {
+        $organizations = [];
+        $this->task("Retrieving your organizations on Fly.io", function() use (&$organizations) {
+            $response = Http::withToken($this->getAuthToken())
+                ->acceptJson()
+                ->contentType("application/json")
+                ->post('https://api.fly.io/graphql', ["query" => "query {currentUser {email} organizations {nodes{id slug name type viewerRole}}}"]);
+
+            // organizations will be an array of arrays that look like this: array("id" => , "slug" => , "name" => , "type" => , "viewerRole" => )
+            $organizations = $response->collect("data.organizations.nodes")->toArray();
+
+            return $response->successful();
+        });
+
+        $organizationNames = [];
+        foreach($organizations as $organization)
+        {
+            $organizationNames[] = $organization["type"] == "PERSONAL" ? "Personal" : $organization["name"];
```

Comment:
Here's what happening (and probably confusing you): 
The menu where users can select what organization they want to deploy in, takes an array with only titles. Weirdly enough the personal organization (which is called 'personal' in every doc) has <your name> for a title, so I made sure it shows up as 'Personal' instead of 'Johannes Werbrouck'.  When the user selects an organization, I figure out which one they picked and return the slug later on.

I do think you've uncovered a bug though! If there's only one organization, the organization's name is returned instead of the slug: 

```
if (sizeOf($organizationNames) == 1)
{
    $this->line("Auto-selected '$organizationNames[0]' since it is the only organization found on Fly.io .");
    return $organizationNames[0];
}
```

I'll update this right away so it return the slug instead of the name!